src/lib/rest/httpRequestSend.cpp

```diff
@@ -258,7 +251,7 @@ int httpRequestSendWithCurl
    const std::string&                         orig_content_type,
    const std::string&                         content,
    const std::string&                         fiwareCorrelation,
-   const std::string&                         ngisv2AttrFormat,
```

Comment:
Is this an actual change? Mabye another TAB should be replace by space indents here...

Reply:
Yes it is. Note old **ngis** vs fixed one **ngsi**

Reply:
Ah, ok, I didn't realized on the typo in the original code. Nice :)

NTC